src/pages/GameMasterView.js

```diff
@@ -143,6 +145,9 @@ const GameMasterView = () => {
     const handleUndoRevive = async (revivedPlayerName) => {
```

Comment:
Just to check that I'm following what's going on here, this function is here because we're eventually adding an undo button? Or is there another use case here?

Reply:
The function is to uncheck a player for a revival mission. We have it to where checking off a player and saving it for a revive mission updates the target mapping, but undoing a checkmark did not undo it. This allows for the unmapping to occur.

Reply:
Hmm wait, so just clicking the checkmark already updates the target mapping?

Reply:
Checking itself won't update, but pressing the 'save' button will. The function is a safety just in case the GM saves a player unintentionally, and wants to revert the changes.

Reply:
Ahh ok good, that sounds good then